server.go

```diff
@@ -24,5 +24,5 @@ func main() {
 
 	n := negroni.Classic()
 	n.UseHandler(router)
-	n.Run(":3000")
+	n.Run(":3001")
```

Comment:
This had to happen because Gin and Negroni were fighting over the same ports in the Vagrant box.